parlant-conversational-agent/loan_approval.py

```
@@ -0,0 +1,189 @@
+import asyncio
+import parlant.sdk as p
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+@p.tool
+async def check_eligibility(context: p.ToolContext, credit_score: int, income: float, loan_amount: float) -> p.ToolResult:
```

Ruff (0.12.2)

Unused function argument: `context`

(ARG001)